components/autofill/src/sync/address/incoming.rs

```diff
@@ -102,10 +114,11 @@ fn save_incoming_records(
 /// incoming changes for the syncing autofill address records.
 fn save_incoming_tombstones(
     conn: &Connection,
-    incoming_tombstones: Vec<Record>,
+    incoming_tombstones: Vec<AddressRecord>,
     signal: &dyn Interruptee,
 ) -> Result<()> {
-    let chunk_size = 1;
+    log::info!("staging {} incoming tombstones", incoming_tombstones.len());
+    let chunk_size = 1; // XXX - chunk_size of 1 seems wrong?
```

Comment:
Assuming this is okay because of [this test case](https://github.com/mozilla/application-services/blob/main/components/support/sql/src/repeat.rs#L103), but also agree this could be refactored. At minimum, the division on line 124 (`sql_support::default_max_variable_number()~ / chunk_size~`) is unnecessary.